Diagnostic/diagnostic.py

```diff
@@ -738,6 +741,13 @@ def start_mdsd():
         eventhub_persist_dir_path).split(" ")
 
     try:
+        # Create monitor object that encapsulates monitoring activities
+        watcher = watcherutil.Watcher(hutil._err, hutil._log)
```

Comment:
Sorry, just saw this... I don't see hutil._err, but see only hutil._error. I wonder if you meant hutil._error here? FYI, pasting HandlerUtility __init__() here:

class HandlerUtility:
    def __init__(self, log, error, s_name=None, l_name=None, extension_version=None):
        self._log = log
        self._error = error

Below is how hutil is instantiated. Not sure if it's to the VM console, though...

waagent.LoggerInit('/var/log/waagent.log','/dev/stdout')
waagent.Log("LinuxAzureDiagnostic started to handle.")
hutil = Util.HandlerUtility(waagent.Log, waagent.Error)